frontend/src/components/common/NavigationBar.js

```diff
@@ -40,15 +41,32 @@ const NavigationBar = ({ user, history, logout }) => {
     }
   }
 
-  let loggedIn = user && user.user ? <AccountCircle /> : ''
-  let username =
-    user && user.user ? (
+  let groupname =
+    group && group.groupName
+      ? (
+        <h4 className="navigation-bar-groupname tracking-in-expand" data-cy='groupname_display_assigned' >
```

Comment:
Is there a reason for changing between double and single quotes? I suggest sticking with double quotes to fit the majority of the code.

Also applies to line 52.